config/sample.yml

```diff
@@ -42,6 +42,16 @@ messageStore: # Postgresql database configuration for message store
   password:
   url:
 
+contactDiscovery: # Configuration for interfacing with Contact Discovery Service cluster
+  accessKey:      # AWS SQS accessKey
+  accessSecret:   # AWS SQS accessSecret
+  queueUrl:       # AWS SQS queue url
```

Comment:
The above three seem like they could be a part of an SQS object

Reply:
yep. will change.

Reply:
done in 8811e29bd3e00e8c272d7650b9abb3622eb13089

Reply:
doesn't look like this file has changed though?

Reply:
correct. this was an oversight.
done in 10d6085688d153137f213bd7354a9ef1510fbece